mantidproject/mantid
# Remove c4250 warning suppression on MSVC

Description of work.
This PR removes the warning suppression when including `<Poco/XML/XMLWriter.h>` - the link ( `http://connect.microsoft.com/VisualStudio/feedback/details/733720/inheriting-from-std-fstream-produces-c4250-warning` ) is dead (tested from 2 different MS accounts) and there is little online except for existing code which suppresses the warning.
On testing locally it doesn't emit additional warnings so it should now be safe to remove these.

Additionally I've reordered the headers to match the order we use across the project. 

**To test:**
Check Windows build - There should be no additional warnings emitted.
Code review

Fixes: No associated issue.

*Does not need to be in the release notes.*

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
